src/main/scala/firrtl/passes/memlib/ReplaceMemTransform.scala

```diff
@@ -138,13 +141,7 @@ class ReplSeqMem extends Transform with HasShellOptions with DependencyAPIMigrat
       new ResolveMemoryReference,
       new CreateMemoryAnnotations(inConfigFile),
       new ReplaceMemMacros(outConfigFile),
-      new WiringTransform,
-      new SimpleMidTransform(RemoveEmpty),
```

Comment:
Is it fine to nix `RemoveEmpty` here?

Reply:
It's still done implicitly by `Block.map` 🤷

Reply:
Awesome. Thanks for taking the time to understand what's really needed here.